tests/functional/object/mcg/ui/test_mcg_ui.py

```
@@ -671,3 +681,185 @@ def test_bucket_pagination(self, setup_ui_class_factory, mcg_obj):
+
+        # 4. Verify the object key is listed in the object browser
+        logger.test_step("Verify object key is listed in the object browser")
+        file_locator = format_locator(
+            bucket_ui.bucket_tab["file_name_text"], object_key
+        )
+        elements = bucket_ui.get_elements(file_locator)
+        assert elements, (
+            f"Object '{object_key}' not found in object browser "
+            f"for bucket '{bucket_name}'"
+        )
+        logger.info(f"Object '{object_key}' is visible in the object browser")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)